elements/Announcer/Speech.test.js

```diff
@@ -221,7 +222,7 @@ describe('Speech', () => {
     speakError = 'network';
     const warnSpy = jest.spyOn(console, 'warn').mockImplementation(() => {});
     jest.useFakeTimers();
-    const speaking = Speech(['Hello', 'World']);
+    const speaking = Speech(['Hello', 'World']); // eslint-disable-line no-unused-vars
```

Comment:
I think this also would work, and follow lint rules, by invoking `Speech` and not assigning the return to a variable.
```js
Speech(['Hello', 'World']);
```

Reply:
I'll give that a try!